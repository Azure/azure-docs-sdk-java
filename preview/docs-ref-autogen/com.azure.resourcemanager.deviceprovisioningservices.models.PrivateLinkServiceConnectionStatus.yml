### YamlMime:JavaType
uid: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus"
fullName: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.deviceprovisioningservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-deviceprovisioningservices:1.1.0-beta.1
