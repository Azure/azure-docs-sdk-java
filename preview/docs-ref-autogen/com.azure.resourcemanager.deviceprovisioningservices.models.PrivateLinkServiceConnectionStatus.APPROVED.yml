### YamlMime:JavaMember
uid: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.deviceprovisioningservices.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.deviceprovisioningservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-deviceprovisioningservices:1.1.0-beta.1
